[2024-12-10T08:43:31.586+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-10T08:43:32.636+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fist_dag_v5.third_task scheduled__2024-12-06T00:00:00+00:00 [queued]>
[2024-12-10T08:43:32.966+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fist_dag_v5.third_task scheduled__2024-12-06T00:00:00+00:00 [queued]>
[2024-12-10T08:43:33.038+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-10T08:43:33.407+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): third_task> on 2024-12-06 00:00:00+00:00
[2024-12-10T08:43:33.445+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=334) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-10T08:43:33.469+0000] {standard_task_runner.py:72} INFO - Started process 351 to run task
[2024-12-10T08:43:33.470+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fist_dag_v5', 'third_task', 'scheduled__2024-12-06T00:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpb2g68ca6']
[2024-12-10T08:43:33.484+0000] {standard_task_runner.py:105} INFO - Job 70: Subtask third_task
[2024-12-10T08:43:33.774+0000] {task_command.py:467} INFO - Running <TaskInstance: fist_dag_v5.third_task scheduled__2024-12-06T00:00:00+00:00 [running]> on host ba06481f48a9
[2024-12-10T08:43:34.369+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='fist_dag_v5' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-06T00:00:00+00:00'
[2024-12-10T08:43:34.379+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-10T08:43:34.454+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-10T08:43:34.471+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello, i am task3 and will be running after task1!']
[2024-12-10T08:43:34.551+0000] {subprocess.py:99} INFO - Output:
[2024-12-10T08:43:34.566+0000] {subprocess.py:106} INFO - hello, i am task3 and will be running after task1!
[2024-12-10T08:43:34.583+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-10T08:43:34.870+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-10T08:43:34.895+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fist_dag_v5, task_id=third_task, run_id=scheduled__2024-12-06T00:00:00+00:00, execution_date=20241206T000000, start_date=20241210T084332, end_date=20241210T084334
[2024-12-10T08:43:35.101+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-10T08:43:35.315+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-10T08:43:35.380+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
